examples/cluster.yaml

```diff
@@ -72,9 +72,9 @@ spec:
       publicSubnetCidr: "10.0.128.0/20"
 
     masters:
-    - imageid: "ami-d60ad6b9"
-      instancetype: "t2.medium"
+    - imageid: "ami-33df775c"
+      instancetype: "m3.medium"
```

Comment:
Minor but t2.medium no longer works because of the EBS volume we mount for docker.